ethereum/contracts/relayer/coreRelayer/CoreRelayer.sol

```diff
@@ -90,14 +74,12 @@ abstract contract CoreRelayer is CoreRelayerDelivery {
      *  If receiverValue >= quoteReceiverValue(targetChain, targetAmount, getDefaultRelayProvider()), then at least 'targetAmount' of targetChain currency will be passed into the 'receiveWormholeFunction' as value.
      *  @param payload an arbitrary payload which will be sent to the receiver contract.
      *  @param vaaKeys Array of VaaKey structs identifying each message to be relayed. Each VaaKey struct specifies a wormhole message, either by the VAA hash, or by the (chain id, emitter address, sequence number) triple.
-     *  The relay provider will call receiveWormholeMessages with an array of signed VAAs specified by this vaaKeys array.
-     *  Specifically, the 'signedVaas' array will have the same length as 'vaaKeys', and additionally for each 0 <= i < vaaKeys.length, signedVaas[i] will match the description in vaaKeys[i]
```

Comment:
It's sufficient to say the signed vaas are provided as specified by the vaakey array.